api/README.md

```diff
@@ -0,0 +1,68 @@
+# To Do API
```

Comment:
@owitiA  are u done with this ?

Reply:
the documentation is the first part and i am working on the API now

Reply:
done